services/ui-src/src/forms/mcpar/mcpar.json

```
@@ -719,31 +719,31 @@
                   "hint": "For what purposes does the state use encounter data collected from managed care plans (MCPs)? Select one or more.<br/>Federal regulations require that states, through their contracts with MCPs, collect and maintain sufficient enrollee encounter data to identify the provider who delivers any item(s) or service(s) to enrollees (42 CFR 438.242(c)(1)).",
                   "choices": [
                     {
-                      "name": "option1",
+                      "name": "rate-setting",
                       "label": "Rate setting"
                     },
                     {
-                      "name": "option2",
+                      "name": "quality/performance-measurement",
```

We might want to standardize our naming to using just dashes, I worry slashes might cause some trouble in the future.

```suggestion
                      "name": "quality-performance-measurement",
```